packages/website/docs/_samples/main/Table/Selection/sample.html

```diff
@@ -12,7 +12,7 @@
     <div class="section">
 <!-- playground-fold-end -->
 		<div id="selectionGroup" role="radiogroup">
-			<ui5-radio-button name="selection" text="Multi" checked></ui5-radio-button>
+			<ui5-radio-button name="selection" text="Multiple" checked></ui5-radio-button>
```

Comment:
@ilhan007 switching selection mode was working fine but it seems like we forgot to change to the new convention value in the test page. 
@pskelin this is a good example not to fallback to the default value of enums, so +1 to 
https://github.com/SAP/ui5-webcomponents/pull/8846#discussion_r1648744480